README.md

```diff
@@ -145,10 +145,37 @@ $ cat /tmp/20200710182129_ksvc_creation_time.csv
   ktest-109,ktest-1,13,13,2,0,0,0,0,15,2,0,2,15
   ktest-110,ktest-1,14,13,1,0,0,0,0,18,3,0,3,18
 ```
-**Sample Dashboard**
+
+### As a Knative developer, I want to have a kperf dashboard to visualize the measurement result
```

Comment:
Suggest changing the usage title from "user story style" to something like "Analyze load test result through Dashboard", I'm also updating other title in this commit https://github.com/knative-sandbox/kperf/pull/23/files